src/nvim/api/ui.c

```diff
@@ -270,6 +292,10 @@ static void remote_ui_grid_resize(UI *ui, Integer grid,
   Array args = ARRAY_DICT_INIT;
   if (ui->ui_ext[kUINewgrid]) {
     ADD(args, INTEGER_OBJ(grid));
+  } else if (grid != 1) {
```

Comment:
do we still need this special case? I would think screen.c by itself will/should avoid inner resizes if multigrid is not active. (But If we still do, the condition should be `!ext_multigrid` and not only `!ext_newgrid` )

Reply:
Also, in `doc/ui.txt`, we have `grid_resize` under `ext_newgrid`. If `ext_newgrid` will be set when `ext_multigrid` is set so does that not cover it?

Reply:
Yes, I got that. What I meant was that `grid_resize` is listed in `ext_newgrid` functions in `ui.txt`. So, I was unsure if the check should be `ext_newgrid` or `ext_multigrid`. You said it should be `!ext_multigrid` and _not only_ `!ext_newgrid` hence the confusion. Sorry... :sweat:

Reply:
> You said it should be !ext_multigrid and not only !ext_newgrid hence the confusion

That would be if we would keep the check (which we should not), sorry for the confusion.